src/VerifyOTP.tsx

```diff
@@ -102,8 +102,16 @@ export default class VerifyOTP extends React.PureComponent<Props, State> {
     }
   }
 
+  invokeOnCountDown() {
+    const { onCountDown } = this.props;
+    if (onCountDown != null) {
+      onCountDown(this.state.countDownSecond);
+    }
+  }
+
   countDown = () => {
     this.setState({ countDownSecond: this.props.countDownFrom });
+    this.invokeOnCountDown();
```

Comment:
It is better to write 

```
this.setState({ countDownSecond: this.props.countDownFrom, () => {
  this.invokeOnCountDown();
});
```

so we do not need to care about whether `setState` is async or not.

Reply:
FYI

https://reactjs.org/docs/faq-state.html#when-is-setstate-asynchronous

Reply:
Didn't know that, thanks. Updated.